site/docs/hosting/vps.md

```diff
@@ -2,9 +2,84 @@
 
 A virtual private server, mostly known as VPS, is a virtual machine running in the cloud with its users having the full control of its system.
 
-In this guide, you'll learn about various methods of running your bot in a VPS, keeping it online 24/7, making it run automatically when your VPS boots and restart on crashes.
+## Preparing Your Code
 
-## systemd
+To get started, you can use the bot shown in the [example of using a session](../plugins/session.md#example-usage).
```

Comment:
Why this example? That makes little sense to me. We might as well use the one on `/`?

Reply:
Why do you need a complex example, though? There's a lot of interesting content here already, so it seems like the natural thing to do is to reduce the complexity of the bot itself as much as possible

Reply:
Initially, I wanted to use a full-fledged control panel, but due to @deptyped's comments, I decided to abandon this idea because otherwise the article would be very long and contain a lot of unnecessary material, and the panel itself would entail many unnecessary modules that are unnecessary for the user.

Reply:
I agree with @deptyped, it makes sense to stay focused in such guides. We should explain how to host a bot on a VPS, and only that. Consequently, it would make sense to use a primitive hello world bot here.